config/default.yml

```diff
@@ -13,3 +13,9 @@ Minitest/RefuteNil:
   StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#refute-nil'
   Enabled: true
   VersionAdded: '0.2'
+
+Minitest/AssertTruthy:
+  Description: 'Check if your test uses `assert(actual)` instead of `assert_equal(true, actual)`.'
+  StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-truthy'
+  Enabled: true
+  VersionAdded: '0.2'
```

Comment:
Cop settings should be arranged in ASCII order. Please move after `Minitest/AssertNil` cop.

Reply:
@koic Fixed.